awx_collection/test/awx/test_completeness.py

```diff
@@ -38,7 +38,7 @@
     'tower_workflow_launch',
     'tower_job_cancel',
     'tower_workflow_template',
-    'tower_ad_hoc_command_wait',
+    'tower_workflow_job_template_schema' 'tower_ad_hoc_command_wait',
```

Comment:
Looks like there's a missing `,` and newline after `'tower_workflow_job_template_schema'`